packages/core/ios/Sources/DdSdkConfiguration.swift

```diff
@@ -62,5 +64,25 @@ public class DdSdkConfiguration: NSObject {
         self.telemetrySampleRate = telemetrySampleRate
         self.vitalsUpdateFrequency = vitalsUpdateFrequency
         self.additionalConfig = additionalConfig
+        self.configurationForTelemetry = configurationForTelemetry
+    }
+}
+
+public class ConfigurationForTelemetry: NSObject {
+    public var initializationType: NSString
+    public var trackErrors: Bool
+    public var trackInteractions: Bool
+    public var trackNetworkRequests: Bool
+    
+    public init(
+        initializationType: NSString?,
+        trackErrors: Bool?,
+        trackInteractions: Bool?,
+        trackNetworkRequests: Bool?
+    ) {
+        self.initializationType = initializationType ?? "LEGACY"
```

Comment:
we don't have fallbacks in the Kotlin counterpart, should we align?

Reply:
I agree we have to align on both platforms. 
Defaults will in theory never be used, so I'd rather remove them, I think this will help us spot more bugs.